### PR TITLE
Jhalackova cockpit fix

### PR DESCRIPTION
**Pull request by lvicoun**
### Description

Adding DC files, replacing version with condition.

### Are there any relevant issues/feature requests?

* bsc#...
* jsc#SLE-...

### Is this (based on) existing content?

* old xml:id...
* old URL...

### CODE REVIEW

File: DC-Micro-6.0-public-cloud [hunk 1]
```diff
@@ -0,0 +1,17 @@
+## ---------------------------- 
+## Doc Config File for the DB Assembly test
+## ----------------------------
+##
+## Basics
+MAIN="SLE-Micro-public-cloud.asm.xml"
+SRC_DIR="articles"
+
```
Comment (tbazant) [inline DC-Micro-6.0-public-cloud:8]: no `IMG_SRC_DIR`?

File: DC-SLE-Micro-5.4-public-cloud [hunk 2]
```diff
@@ -0,0 +1,17 @@
+## ---------------------------- 
+## Doc Config File for the DB Assembly test
+## ----------------------------
+##
+## Basics
+MAIN="SLE-Micro-public-cloud.asm.xml"
+SRC_DIR="articles"
+
```
Comment (tbazant) [inline DC-SLE-Micro-5.4-public-cloud:8]: no `IMG_SRC_DIR`?

File: glues/cockpit-user-management.xml [hunk 3]
```diff
@@ -43,7 +43,7 @@
         <xref linkend="task-creating-users-cockpit"/>
       </para>
     </listitem>
-    <listitem>
+    <listitem os="slemicro;slmicro" condition="5.5;6.0">
```
Comment (tbazant) [inline glues/cockpit-user-management.xml:46]: not sure this will work as expected...

**Review by tbazant — COMMENTED** [review]
small nits only

File: DC-Micro-6.0-public-cloud (hunk 1 above)
Comment (lvicoun) [reply DC-Micro-6.0-public-cloud:8]: No images there.

File: glues/cockpit-user-management.xml (hunk 3 above)
Comment (lvicoun) [reply glues/cockpit-user-management.xml:46]: The condition? I'm sure that os works this way.

File: articles/cockpit-slemicro.asm.xml [hunk 4]
```diff
@@ -507,4 +506,325 @@ type="application/relax-ng-compact-syntax"?>
       <output renderas="appendix"/>
     </module>
   </structure>
+
+  <structure renderas="article" xml:id="slemicro-cockpit-54">
+    <merge>
+      <title>Administering &productname; using &cockpit;</title>
+      <revhistory xml:id="rh-slemicro-cockpit-54">
+        <revision><date>2023-05-15</date>
```
Comment (tomschr) [inline articles/cockpit-slemicro.asm.xml:514]: Is the date still correct?

File: articles/SLE_Micro_admin.asm.xml [hunk 5]
```diff
@@ -200,8 +200,8 @@
         <phrase>&power;</phrase>
       </meta>
       <meta name="productname" its:translate="no">
-        <productname version="5.4" os="slemicro">&productname;</productname>
-        <productname version="5.5" os="slmicro">&productname;</productname>
+        <productname condition="5.4" os="slemicro">&productname;</productname>
+        <productname condition="5.5" os="slmicro">&productname;</productname>
```
Comment (tomschr) [inline articles/SLE_Micro_admin.asm.xml:204]: Maybe simplify it?

```suggestion
        <productname os="slemicro">&productname;</productname>
```

File: articles/SLE-Micro-public-cloud.asm.xml [hunk 6]
```diff
@@ -67,9 +67,9 @@
       </meta>
       <!-- enter product name and version -->
       <meta name="productname" its:translate="no">
-        <productname version="5.4" os="slemicro">&productname;</productname>
-        <productname version="5.5" os="slemicro">&productname;</productname>
-        <productname version="6.0" os="slmicro">&productname;</productname>
+        <productname condition="5.4" os="slemicro">&productname;</productname>
+        <productname condition="5.5" os="slemicro">&productname;</productname>
+        <productname condition="6.0" os="slmicro">&productname;</productname>
```
Comment (tomschr) [inline articles/SLE-Micro-public-cloud.asm.xml:72]: It's technically correct, but IMHO a bit too verbose. It's up to you if you want to change it, but I would simplify it. The `condition` attribute here is not really necessary, as the content (`productname` entity) is always the same. The entity contains already the correct title, correct?

As such, you could simplify it:

```suggestion
        <productname os="slemicro">&productname;</productname>
```

File: articles/cockpit-slemicro.asm.xml [hunk 7]
```diff
@@ -182,9 +182,8 @@ type="application/relax-ng-compact-syntax"?>
       </meta>
       <!-- enter product name and version -->
       <meta name="productname" its:translate="no">
-        <productname version="5.4" os="slemicro">&productname;</productname>
-        <productname version="5.5" os="slemicro">&productname;</productname>
-        <productname version="6.0" os="slmicro">&productname;</productname>
+        <productname condition="5.5" os="slemicro">&productname;</productname>
+        <productname condition="6.0" os="slmicro">&productname;</productname>
```
Comment (tomschr) [inline articles/cockpit-slemicro.asm.xml:186]: Same here:

```suggestion
        <productname os="slemicro">&productname;</productname>
```

File: articles/cockpit-slemicro.asm.xml [hunk 8]
```diff
@@ -507,4 +506,325 @@ type="application/relax-ng-compact-syntax"?>
...
+      <!-- enter the platform identifier or list of comma-separated list of identifiers -->
+      <meta name="architecture" its:translate="no">
+        <phrase>&x86-64;</phrase>
+        <phrase>&power;</phrase>
+        <phrase>&zseries;</phrase>
+        <phrase>&aarch64;</phrase>
+      </meta>
+      <!-- enter product name and version -->
+      <meta name="productname" its:translate="no">
+        <productname condition="5.4" os="slemicro">&productname;</productname>
```
Comment (tomschr) [inline articles/cockpit-slemicro.asm.xml:535]: Although it's correct to use `condition="5.4"` here, you can also remove the attribute. It's not much of a difference. The structure will be selected anyway if 5.4 is built.

File: articles/cockpit-slemicro.asm.xml [hunk 9]
```diff
@@ -507,4 +506,325 @@ type="application/relax-ng-compact-syntax"?>
...
+          <dm:component>Smart Docs</dm:component>
+          <dm:product>Documentation</dm:product>
+          <dm:assignee>jsindelarova@suse.com</dm:assignee>
+        </dm:bugtracker>
+        <dm:translation>yes</dm:translation>
+      </dm:docmanager>
+      <abstract>
+        <variablelist>
+          <varlistentry>
+            <term>TOPIC</term>
```
Comment (tomschr) [inline articles/cockpit-slemicro.asm.xml:565]: According to Jana's template in [doc-modular/templates/articles/assembly.asm.xml (line 156ff)](https://github.com/SUSE/doc-modular/blob/bd5dab7179361041ebba1a4a5f2c4d75a4bd8af2/templates/articles/assembly.asm.xml#L156) it should be "WHAT?", "WHY?", "EFFORT", "GOAL", and "REQUIREMENTS".

File: articles/elemental.asm.xml [hunk 10]
```diff
@@ -86,8 +86,8 @@
         <phrase>&power;</phrase>
       </meta>
       <meta name="productname" its:translate="no">
-        <productname version="5.5" os="slemicro">&productname;</productname>
-        <productname version="6.0" os="slmicro">&productname;</productname>
+        <productname condition="5.5" os="slemicro">&productname;</productname>
+        <productname condition="6.0" os="slmicro">&productname;</productname>
```
Comment (tomschr) [inline articles/elemental.asm.xml:90]: Same here:

```suggestion
        <productname os="slemicro">&productname;</productname>
```

**Review by tomschr — COMMENTED** [review]
Basically it looks fine. I think (although not tested) you could simplify this a lot. In most cases you don't need the condition attribute on the productname. Most is done with the entity productname anyway.

I would only add the condition on productname element if there is a substantial change.

File: glues/cockpit-user-management.xml (hunk 3 above)
Comment (lvicoun) [reply glues/cockpit-user-management.xml:46]: No, slemicro is for 5.X releases as its name is SLE Micro, slmicro is for 6.X with the name SUSE Linux Micro. Both product names needs to coexist in the same repository.

**Pull request merged**